Servers/bare-metal-faq.md

```diff
@@ -101,8 +101,16 @@ No, the storage configurations provisioned for Bare Metal servers are all either
+No, the available Operating Systems include Windows 2012 Standard Edition R2, Windows 2012 Datacenter Edition, RHEL6, CentOS 6, and Ubuntu 14. If there is a particular OS image you would like to see incorporated please [submit a feature request](https://www.ctl.io/knowledge-base/support/how-do-i-submit-a-feature-request/).
 
 **Can I use my own licensing for the OS?**
 
-No, the licensing is factored into the cost of the Bare Metal server where applicable and can not be separated from the service.  If you would be interested in using your own OS licensing with Bare Metal servers, please let us know in a [feature request](https://www.ctl.io/knowledge-base/support/how-do-i-submit-a-feature-request/).
+No, the licensing is factored into the cost of the Bare Metal server where applicable and cannot be separated from the service.  If you would be interested in using your own OS licensing with Bare Metal servers, please let us know in a [feature request](https://www.ctl.io/knowledge-base/support/how-do-i-submit-a-feature-request/).
+
+**What's the difference between Windows 2012 Standard and Windows 2012 Datacenter Edition on Bare Metal servers?**
+
+Both are capable of leveraging the Hyper-V role to provision VMs on the Bare Metal server, but the Datacenter Edition license allows for unlimited VMs to be provisioned on a physical host while the Standard Edition is limited to two VMs running concurrently on the physical server host.
+
+**How do I activate my Windows VMs provisioned on a Windows 2012 Datacenter Edition Bare Metal server?**
+
+The answers will differ depending on which Windows OSes you are trying to activate.  Guest VMs running Windows 2012 R2 will automatically activate using [AVMA (Automatic Virtual Machine Activation)](https://technet.microsoft.com/en-us/library/dn303421.aspx) using the host's Hypervisor. Guest VMs running Windows 2008 R2 can activate either using Multi-Activation Keys (MAKs) obtained from Microsoft, or by using the volume license keys available on [Microsoft's site](https://technet.microsoft.com/en-us/library/jj612867.aspx?f=255&MSPPError=-2147217396) along with the CenturyLink KMS host. To use our KMS you will need to [manually specify the KMS host](https://technet.microsoft.com/en-us/library/ff793406.aspx?f=255&MSPPError=-2147217396) on the guest VM. You will need the VM to be on a CenturyLink Cloud customer VLAN to access our KMS host and the IP/port associated, which is 172.17.1.21:1688.  Windows VMs in a Hyper-V environment on Bare Metal servers should use IPs in the same VLAN as configured in the Control portal for the host Bare Metal server.  It is also recommended that IPs be higher in the range so VMs claiming from that same range will not encounter IP conflicts.  For activation of older Windows OSes on guest VMs you will need to obtain your own MAKs and/or activation solution.
```

Comment:
I think we need to provide the commands for the KMS activation and how to validate ports are open (telnet to that tcp port)

```
slmgr /skms 172.17.1.21:1688
```

make the answer easier.....

you should also detail how a customer can claim additional IP's in Control for the Guest VMs.  Until the blueprint feature exists its basically a NOC ticket but we should have a clear statement on how that is done